roofit/roofitcore/src/RooFuncWrapper.cxx

```diff
@@ -247,7 +250,7 @@ void RooFuncWrapper::writeDebugMacro(std::string const &filename) const
    // Remove duplicated declared functions
    for (std::string const &name : _collectedFunctions) {
       if (seenFunctions.count(name) > 0) {
-         continue;
+         continue; // needed in order to set the index to constant in the RooMultiPdf
```

Comment:
```suggestion
         continue;
```
I think you put the comment in the wrong line. Maybe you wanted to add this to line 149?